BankDemo.py

```diff
@@ -1,129 +1,160 @@
 """"importing modules"""
-from flask import Flask, redirect, url_for, jsonify
-import pyodbc
+import  json
+from flask import Flask, redirect, url_for, abort, request
+import DBconnection
+import DBqueries
 
 
-app = Flask(__name__)
+# Calling Flask API
+APP = Flask(__name__)
 
 
-def init_db():
-    """"connecting to database"""
-    server = '192.168.18.36'
-    db = 'HBK_Test'
-    user = 'kailashp'
-    password = 'Welcome@731'
-    con = pyodbc.connect('Driver={ODBC Driver 17 for SQL Server};'
-                         'SERVER=' + server + ';DATABASE=' + db +
-                         ';UID=' + user + ';PWD=' + password)
-    cursor = con.cursor()
-    return cursor
+CURSOR = DBconnection.init_db()
 
 
-cursor = init_db()
-
-
-@app.route('/login/<userID>,<password>')
+@APP.route('/login/<userID>,<password>', methods=['GET', 'POST'])
 def login(userID, password):
     """"checking login details"""
-    login_object = get_login_details(userID, password)
-    if type(login_object) is pyodbc.Row:
-        return redirect(url_for('accountdetails', userid=userID))
-    return "UserID or password is invalid"
+    try:
+        login_object = get_login_details(userID, password)
+        if login_object:
+            return redirect(url_for('accountdetails', userid=userID, password=password))
```

Comment:
instead of redirecting, why can't you make a login decorator. code will be cleaner.